org.eclipse.winery.tools.deployablecomponents/src/main/java/org/eclipse/winery/tools/deployablecomponents/DeployableComponents.java

```diff
@@ -0,0 +1,132 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache Software License 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *******************************************************************************/
+
+package org.eclipse.winery.tools.deployablecomponents;
+
+import javafx.util.Pair;
+import org.eclipse.winery.model.tosca.TNodeType;
+import org.eclipse.winery.tools.deployablecomponents.commons.Component;
+import org.eclipse.winery.tools.deployablecomponents.commons.Dockerfile;
+import org.eclipse.winery.tools.deployablecomponents.crawler.Crawler;
+import org.eclipse.winery.tools.deployablecomponents.crawler.CrawlerType;
+import org.eclipse.winery.tools.deployablecomponents.fileanalyzer.Fileanalyzer;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+/* Example how to use:
```

Comment:
Please write a documentation about the code in the `docs` folder and remove it here.